fivox/compartmentLoader.cpp

```diff
@@ -49,8 +49,6 @@ class CompartmentLoader::Impl
         , _report( _config.getReportSource( params.getReport( )),
                    brion::MODE_READ, _target)
     {
-        _report.updateMapping( _target );
-
```

Comment:
Even if the report target and _target don't match? Why does this function exist?